### DISCUSSION TITLE
Free text/markup field

### DISCUSSION THREAD

**Discussion by gazm2k5** [nextcloud/cookbook#2349] [Feedback]
I like the look and functionality of the app, addings tags, ingredients etc, but a free text field, or even better a mark up field would be perfect.

All my existing recipes (which currently live on Notion) are a mix of formats. Some of them will list 5 ingredients for the sauce and then a paragraph on method, then 5 more for the main dish etc. After that might be some alternative methods or ingredients to make the dish differently.

There is currently no way to just have recipes written free style. Not all my recipes have a rigid list of ingredients and steps to follow in chronological order.

I have no way of moving my recipes from notion to Cookbook as there is no field to copy and paste them into. I tried using the "Description" field but there are no line breaks so it's extremely difficult to read.

![image](https://github.com/nextcloud/cookbook/assets/43571579/feaa1b08-bef4-4683-82c0-bf8963df028b)
See an example of a mayonnaise recipe that also has an extra part to turn the mayonnaise into a patatas bravas sauce.

**Comment by christianlupus** [comment]
The description field (as well as others) do provide markdown support. Would you please check if an empty line between paragraphs helps you?

**Reply by gazm2k5** [reply]
Ah so it does! Yes, 2 line breaks gives an extra line. And 3 line breaks gives a paragraph. It's a little bit faffy to have to add extra line breaks to all my recipes as I copy them over from notion but adding bullet points is nice.

Thanks!
